resources/waterheater.rb

```diff
@@ -9,7 +9,7 @@
 
 class Waterheater
   def self.apply_tank(model, runner, space, fuel_type, cap, vol, ef,
-                      re, t_set, oncycle_p, offcycle_p, ec_adj, nbeds, dhw_map, sys_id)
+                      re, t_set, oncycle_p, offcycle_p, ec_adj, nbeds, dhw_map, sys_id, has_desuperhater, related_clg_coil)
```

Comment:
"desuperhater" -> "desuperheater"

I hope this doesn't mean you're subconsciously hating desuperheaters! 🤣

Reply:
Hahahahahaha, I've already fixed some places with this misspelling, one more left out!